openshift/system/Dockerfile.on_prem

```diff
@@ -58,6 +59,8 @@ RUN export ${BUNDLER_ENV} >/dev/null \
     && chmod g+w -vfR log tmp public/assets db/sphinx \
     && umask 0002 \
     && cd /opt/system \
+    && npm set progress false \
+    && npm install \
```

Comment:
` npm install --no-progress` (?)

Reply:
Saw the other comment... does is it set CI env then?

Reply:
It must be set in our circleci build but this commands are executed inside the docker container, don't they?
In any case it looks better with `--no-progress`.